s3fs/core.py

```diff
@@ -1351,14 +1394,23 @@ async def _copy_managed(self, path1, path2, size, block=5 * 2 ** 30, **kwargs):
         )
         self.invalidate_cache(path2)
 
-    async def _cp_file(self, path1, path2, **kwargs):
-        gb5 = 5 * 2 ** 30
+    async def _cp_file(self, path1, path2, preserve_etag=None, **kwargs):
+        threshold = kwargs.pop("managed_threshold", 5 * 2 ** 30)
```

Comment:
This is mostly for testing, since I didn't want to create a giant 5GB file on the memory and write it somewhere on the disk.

Reply:
It feels like this should be a module-level constant, so you can easily change it with a pytest fixture.